### PR TITLE
apply focus to Select component

### PR DESCRIPTION
**Pull request by hokakede1**

### CODE REVIEW

**Comment by sambev** [comment]
Did you run an npm update? Seems weird for you to have a ton of package-lock file changes with this.

File: src/components/Select.js [hunk 1]
```diff
@@ -240,7 +241,11 @@ class Select extends React.Component {
           fontSize: theme.FontSizes.MEDIUM,
           padding: '8px 10px',
           position: 'relative',
-          boxSizing: 'border-box'
+          boxSizing: 'border-box',
+          ':focus': {
+            outline: 'dotted thin #333',
```
Comment (sambev) [inline src/components/Select.js]: This should probably use a constant instead of #333. Maybe GRAY_900?

**Comment by hokakede1** [comment]
@sambev I restore the old package-lock and apply GRAY_900

**Comment by cerinman** [comment]
:shipit:

**Pull request merged**